mosquitto_bridge_registrator.py

```diff
@@ -81,6 +80,18 @@ def __init__(self, system_id, restart_mosquitto=True):
 		self._restart_mosquitto = restart_mosquitto
 		self._requests_log_level = logging.getLogger("requests").getEffectiveLevel()
 
+	def _get_vrm_broker_url(self):
+		"""To allow scaling, the VRM broker URL is generated based on the system identifier
+		The function returns a numbered broker URL between 0 and VrmNumberOfBrokers, which makes sure
+		that broker connections are distributed equally between all VRM brokers
+		"""
+		sum = 0
+		for character in self._system_id:
+			sum += ord(character)
```

Comment:
Should we be worried about the assumed encoding? This code assumes the source is unicode. If you do this on another platform, it happens to work for ASCII as long there are no special chars, but it's somewhat undefined. And I hope we don't ever see EBCDIC...

Also, we should do a `lower()` and `strip()` for safety.

Reply:
Done!